bin/src/run/watch.ts

```diff
@@ -159,12 +159,6 @@ export default function watch(
 	const removeOnExit = onExit(close);
 	process.on('uncaughtException', close);
 
-	// only listen to stdin if it is a pipe
-	if (!process.stdin.isTTY) {
-		process.stdin.on('end', close); // in case we ever support stdin!
-		process.stdin.resume();
-	}
```

Comment:
Is this a bug fix, independent of the above? Or are both changes needed for the lerna support?